pkg/controller/queuejobresources/pod/pod.go

```diff
@@ -176,11 +179,12 @@ func (qjrPod *QueueJobResPod) scaleUpQueueJobResource(resIndex int, diff int32,
 		return err
 	}
 
-	if template.Template.Labels == nil {
-
-		template.Template.Labels = map[string]string{}
+	//TODO: need set reference after Pod has been really dded
+	tmpPod := v1.Pod{}
+	qjrPod.refManager.AddReference(qjobRes, &tmpPod)
```

Comment:
suggest adding error handling for `AddReference` when the return value is not nil

Reply:
ok